src/test/g8/giter8.test

```diff
@@ -0,0 +1,15 @@
+# This is an sbt scripted test
+
+> compile
+> test
+
+# TODO: Switch to g8ScaffoldForce and uncomment last two lines when
+# https://github.com/foundweekends/giter8/pull/771 got merged
+> g8Scaffold form --model=user
+
+$exists app/controllers/UserController.java
+$exists app/controllers/UserData.java
+$exists test/controllers/UserControllerTest.java
+
+#> compile
+#> test
```

Comment:
If https://github.com/foundweekends/giter8/pull/771 gets merged we can then even compile and test after applying the form template.

Reply:
Btw, just a reminder, this script name here can have various names: https://github.com/foundweekends/giter8/blob/v0.16.1/plugin/src/main/scala/Giter8Plugin.scala#L152-L161